src/main/scala/ai/privado/languageEngine/java/processor/JavaProcessor.scala

```diff
@@ -158,7 +159,15 @@ object JavaProcessor {
       println(s"${Calendar.getInstance().getTime} - Downloading dependencies and Parsing source code...")
     else
       println(s"${Calendar.getInstance().getTime} - Parsing source code...")
-    cpgconfig = Config(inputPath = sourceRepoLocation, fetchDependencies = !config.skipDownloadDependencies)
+
+
+
+    if (!Files.exists(Paths.get("./.privado"))) {
+      Files.createDirectory(Paths.get("./.privado"));
+    }
+
+    val cpgOutputPath = s"${Paths.get(".").toAbsolutePath}/.privado/cpg.bin"
```

Comment:
@karan-batavia the name `.privado` is already part of Constants, use that so that in future we just have to make change in the constants to change the output folder. Also the name of the binary which is `cpg.bin` store that in Constants as well